### ISSUE TITLE
Naming Graphs of Custom Monitor Data

### ISSUE THREAD

**Issue by InH4te** [naver/ngrinder#202]
Following [this](http://www.cubrid.org/wiki_ngrinder/entry/how-to-add-custom-monitor-data) link, it shows us how to make custom monitor data graphs. But when the appear, they appear as "CUSTOM MONITOR DATA 1", "CUSTOM MONITOR DATA 2" ...

Is there anyway to give custom names to these graphs?

**Comment by songeunwoo** [comment]
hello inH4te.
Current custom name it is fixed. ([link](https://github.com/naver/ngrinder/blob/develop/ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestService.java)) 
we will think of ways you can modify the name of the chart.

**Comment by songeunwoo** [comment]
@junoyoon 
please comment.

**Comment by junoyoon** [comment]
No. it's not possible now...  it's hard-coded in the following location.

https://github.com/naver/ngrinder/blob/develop/ngrinder-controller/src/main/webapp/WEB-INF/ftl/perftest/detail_report/monitor.ftl#L39
